TRADEMARK.md

```diff
@@ -0,0 +1,122 @@
+
+### 3 Here are examples of incorrect logos. The logo should not be altered in any way. Do not add, remove, or resize any elements or colours of the existing logo, Harmony eDelivery Access logo, for example:
+
+|                                                                                                                              |                                                                                                                                          |
+|:----------------------------------------------------------------------------------------------------------------------------:|:----------------------------------------------------------------------------------------------------------------------------------------:|
+|     ![Bad example 1](img/trademark/bad_example_1.jpg)<br>Don’t add outlines, special effects, or other graphic elements.     |            ![Bad example 2](img/trademark/bad_example_2.jpg)<br>Don’t alter or add copy to the logo. Don’t change the fonts.             |
+| ![Bad example 3](img/trademark/bad_example_3.jpg)<br>Make sure the logo is entirely legible and on a contrasting background. |                               ![Bad example 4](img/trademark/bad_example_4.jpg)<br>Don’t distort the logo.                               |
+|                  ![Bad example 5](img/trademark/bad_example_5.jpg)<br>Don’t change or remove logo elements.                  |                        ![Bad example 6](img/trademark/bad_example_6.jpg)<br>Don’t change the colours of the logo.                        |
+|                      ![Bad example 7](img/trademark/bad_example_7.jpg)<br>Don’t put the logo in shape.                       | ![Bad example 8](img/trademark/bad_example_8.jpg)<br>Don’t allow the logo to get cut off or be covered. The entire logo must be visible. |
+
+## 4 Questions and Comments
+
+In case of questions about these guidelines, if you are interested in obtaining a license to use the trademarks, or to
+report concerns regarding the use or misuse of the trademarks of NIIS, contact us at <info@niis.org>.
+
+## 5 Updates of Guidelines
+
+The NIIS® and Harmony eDelivery Access® Trademark Usage Guidelines may be modified from time to time by NIIS
+at our sole discretion and release new versions without notice. The current version is 2025 v1, published in
+January 2025.
+
+## 6 Q&A
+
+1. **Question**: Does this also mean that if I, for example, compile Harmony eDelivery Access® Access Point (without changing any actual code) and create my metapackages to distribute in a country, for instance, I should always modify the frontend code and remove the Harmony eDelivery Access logos?
+    1. **Answer**: In this case, you don’t need to remove the Harmony eDelivery Access logos from the frontend because you’re still using the original Harmony eDelivery Access software. However, the national data exchange service that’s built using Harmony eDelivery Access should be called something else.
```

Comment:
This example is very X-Road specific and therefore, it's better to change it. For example, we could use this instead:

> 1. **Question**: Does this also mean that if I, for example, compile Harmony eDelivery Access® Access Point including my own changes to the source code and publish a modified version of the software for distribution, for instance, I should modify the frontend code and remove the Harmony eDelivery Access logos?
>    1. **Answer**: In this case, you need to remove the Harmony eDelivery Access logos from the frontend because you're not using the original Harmony eDelivery Access software anymore.